flow-shop/chapter.md

```diff
@@ -1,18 +1,20 @@
 
-The objective in flow shop scheduling is to minimize the total time it takes to process all of the tasks from every job to completion. Typically, this objective is referred to as the *makespan*. This general problem has many applications, but is most related to optimizing production facilities for the construction of various commercial products.
+The objective in flow shop scheduling is to minimize the total time it takes to process all of the tasks from every job to completion. Typically, this total time is referred to as the *makespan*. The flow shop scheduling problem has many applications, but is most related to optimizing production facilities.
 
-Every flow shop problem consists of *n* machines and *m* jobs. In our car example, there will be *n* stations to work on the car and *m* cars to make in total. Each job is made up of exactly *n* tasks, and we can assume that the *i*-th task of a job must use machine *i* and requires a predetermined amount of processing time: *p*(*j*,*i*) is the processing time for the *i*-th task of job *j*. Further, the order of the tasks for any given job should follow the order of the machines available -- for a given job, task *i* must be completed prior to the start of task *i*+1. In our car example, we wouldn't want to start painting the car before the frame was assembled. The final restriction is that no two tasks can be processed on a machine simultaneously.
+<!-- FIXME: Might as well make these italics into math symbols at some point-->
+Every flow shop problem consists of *n* machines and *m* jobs. In our car example, there will be *n* stations to work on the car and *m* cars to make in total. Each job is made up of exactly *n* tasks, and we can assume that the *i*-th task of a job must use machine *i* and requires a predetermined amount of processing time: *p*(*j*,*i*) is the processing time for the *i*-th task of job *j*. Further, the order of the tasks for any given job should follow the order of the machines available; for a given job, task *i* must be completed prior to the start of task *i+*1. In our car example, we wouldn't want to start painting the car before the frame was assembled. The final restriction is that no two tasks can be processed on a machine simultaneously.
 
 Because the order of tasks within a job is predetermined, a solution to the flow shop scheduling problem can be represented as a permutation of the jobs. The order of jobs processed on a machine will be the same for every machine, and given a permutation, a task for machine *i* in job *j* is scheduled to be the latest of the following two possibilities:
```

Comment:
No worries! Some other authors have been using MathJax, but @MichaelDiBernardo and I will clean this up.